lib/std/Thread/Condition.zig

```diff
@@ -152,9 +152,9 @@ pub const AtomicCondition = struct {
             @atomicStore(bool, &cond.pending, true, .SeqCst);
         }
 
-        mutex.unlock();
+        mutex.releaseDirect();
```

Comment:
Hi, thanks for taking the time to review the PR. My reasoning was simple (maybe simplistic): since no instance of `Held` is available to me, I just used the `Mutex` API. Would it make sense to extend the interface of `wait` to include the `Held` instance? This would feel repetitive and error prone, but there's probably something I am not seeing.

Reply:
> Would it make sense to extend the interface of wait to include the Held instance?

`Held` is defined in `Mutex.zig`, but not needed as we dont expose it to the user. I asked about `unlock`, but I checked the git log and nothing shows up so it looks like it was broken from the commit.